apiserver/admin.go

```diff
@@ -533,7 +533,14 @@ func (a *admin) maintenanceInProgress() bool {
 	return !a.srv.upgradeComplete()
 }
 
-func setupPingTimeoutDisconnect(clock clock.Clock, root *apiHandler, entity state.Entity) error {
+// PingRootHandler is the interface that the root handler must implement
+// to allow the pinger to be registered.
+type PingRootHandler interface {
```

Comment:
Any reason to export this? It isn't used elsewhere.

Reply:
I was thinking there should be a test for this.